AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientController.swift

```diff
@@ -99,6 +99,14 @@ extension DefaultAudioClientController: AudioClientController {
         eventAnalyticsController.publishEvent(name: .meetingStartRequested)
         let appInfo = DeviceUtils.getAppInfo()
         muteMicAndSpeaker = audioMode == .noAudio
+        var audioModeInternal: AudioModeInternal = .Stereo48K
+        if (audioMode == .mono48K) {
+            audioModeInternal = .Mono48K
+        } else if (audioMode == .mono16K) {
+            audioModeInternal = .Mono16K
+        } else if (audioMode == .noAudio) {
+            audioModeInternal = .NoAudio
```

Comment:
just curious will check-in mode be released with 48K Hz?

Reply:
We may be reverting CheckIn mode if it can't be supported on all platforms before Stereo GA

Reply:
Got it. So `.NoAudio` will be removed.

Reply:
Will update use case in README.md, and put Breaking in Changelog.